server/controllers/authController.api.js

```diff
@@ -25,11 +30,17 @@ exports.login = (req, res) => {
         if (err) {
             return res.status(500).json({ message: 'an error occurred' });
```

Comment:
I know this is not your code, but we shouldn't be returning 500s in the code

Reply:
Probably should be 400

Reply:
I'll change this to a 400 error then